Neos.Eel/Tests/Unit/Helper/ArrayHelperTest.php

```diff
@@ -165,7 +165,8 @@ public function indexOfExamples()
         return [
             'empty array' => [[], 42, null, -1],
             'array with values' => [['a', 'b', 'c', 'b'], 'b', null, 1],
-            'with offset' => [['a', 'b', 'c', 'b'], 'b', 2, 3]
+            'with offset' => [['a', 'b', 'c', 'b'], 'b', 2, 3],
+            'associative' => [['a' => 'el1', 'b' => 'el2'], 'el2', null, 1]
```

Comment:
```suggestion
            'associative' => [['a' => 'el1', 'b' => 'el2'], 'el2', null, 1],
            'associative with offset' => [['a' => 'el1', 'b' => 'el2'], 'el2', 1, 1]
```
This should then also hold to keep in line with non-assoc behaviour.